.storybook/preview.js

```diff
@@ -1,36 +1,16 @@
 import {
-    addParameters,
     addDecorator,
     setCustomElements,
-    withA11y,
-    withKnobs,
-    withWebComponentsKnobs,
-} from '@open-wc/demoing-storybook';
+} from '@web/storybook-prebuilt/web-components.js';
+// import customElemenents from '../custom-elements.json';
```

Comment:
If we're comfortable with the args listing at current can remove this comment and the one at the end of the file? We can also remove the unused import of `setCustomElements`, right?